onnxruntime/core/providers/cpu/reduction/reduction_ops.cc

```diff
@@ -286,6 +286,10 @@ bool PrepareForReduce(const Tensor* input_tensor_ptr,
     return true;
   }
 
+  if (0 == first_dim) {
```

Comment:
Interesting that this came up as an xcode warning?

Reply:
Yes it warns that the var might be zero.

Reply:
Removed.